homeassistant/components/home_connect/api.py

```diff
@@ -156,6 +157,25 @@ def get_program_sensors(self):
         ]
 
 
+class DeviceWithOpState(HomeConnectDevice):
+    """Device that has an operation state sensor."""
+
+    def get_opstate_sensor(self):
+        """Get a dictionary with info about operation state sensor."""
+
+        return [
```

Comment:
The docstring says it will return a dictionary.

The return value is a list and it's later also used to add to another list. 

Think the docstring is wrong?
```suggestion
        """Get a list with info about operation state sensors."""

        return [
```